src/wallet/rpc/spend.cpp

```diff
@@ -71,7 +71,7 @@ static void InterpretFeeEstimationInstructions(const UniValue& conf_target, cons
     } else {
         options.pushKV("fee_rate", fee_rate);
     }
-    if (!options["conf_target"].isNull() && (options["estimate_mode"].isNull() || (options["estimate_mode"].get_str() == "unset"))) {
+    if (!options["conf_target"].isNull() && (options["estimate_mode"].isNull() || (ToLower(options["estimate_mode"].get_str()) == "unset"))) {
         throw JSONRPCError(RPC_INVALID_PARAMETER, "Specify estimate_mode");
     }
```

Comment:
I think we should also return error when `estimatemode` is passed without confirmation target and a a test for the test case ?
Maybe in its own commit or a follow-up PR.
```suggestion
    }
    if (options["conf_target"].isNull() && !options["estimate_mode"].isNull()) {
        throw JSONRPCError(RPC_INVALID_PARAMETER, "estimate_mode should be passed with conf_target");
    }
```